packages/vscode-extension/package.nls.json

```diff
@@ -157,6 +157,30 @@
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorTitle": "Manage Collaborator",
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorDescription": "Manage M365 Teams App (with Microsoft Entra app) Collaborators",
     "teamstoolkit.commandsTreeViewProvider.addWebpartTitle": "Add SPFx Web Part",
+    "teamstoolkit.commandsTreeViewProvider.officeDevDeployTitle": "Deploy",
+    "teamstoolkit.commandsTreeViewProvider.officeDevDeployDescription": "Deploy",
+    "teamstoolkit.commandsTreeViewProvider.publishAppSourceTitle": "Publish",
+    "teamstoolkit.commandsTreeViewProvider.publishAppSourceDescription": "Link to the wiki about how to publish the add-in to AppSource",
+    "teamstoolkit.commandsTreeViewProvider.createOfficeAddInTitle": "Create a New Add-in",
+    "teamstoolkit.commandsTreeViewProvider.createOfficeAddInDescription": "Create a new add-in project of Word Excel or Powerpoint",
+    "teamstoolkit.commandsTreeViewProvider.checkAndInstallDependenciesTitle": "Check and Install Dependencies",
+    "teamstoolkit.commandsTreeViewProvider.checkAndInstallDependenciesDescription": "Check and install dependencies",
+    "teamstoolkit.commandsTreeViewProvider.officeDevLocalDebugTitle": "Preview Your Add-in (F5)",
+    "teamstoolkit.commandsTreeViewProvider.officeDevLocalDebugDescription": "Local debug your Add-in App",
+    "teamstoolkit.commandsTreeViewProvider.validateManifestTitle": "Validate Manifest",
+    "teamstoolkit.commandsTreeViewProvider.validateManifestDescription": "Validate the manifest of Office add-ins project",
```

Comment:
Proposed: Validate the manifest file of Office add-ins project.
Logic: It's clearer and easier to perceive by users.

Reply:
fixed.